lib/versioned/runner.js

```diff
@@ -97,7 +98,19 @@ function installPackages(cb) {
 function runTests(cb) {
   // TODO: Add tap arguments, such as color.
   process.send({ status: 'running' })
-  spawn('node', [testFile], function testHandler(err) {
+  let args = [testFile]
+  if (process.env.packageType && process.env.packageType === 'module') {
```

Comment:
thinking more of this we could just default the type in lib/versioned/test.js to `commonjs`

Reply:
I initially did do that--it's easy enough to restore!